packages/mask/src/site-adaptors/mirror.xyz/utils/selectors.ts

```diff
@@ -14,17 +14,14 @@ export const querySelectorAll = <T extends E>(selector: string) => {
 export const entryInfoSelector: () => LiveSelector<E, true> = () =>
     querySelector<E>('div+button').map((x) => x.parentElement?.firstElementChild?.lastElementChild as HTMLElement)
 
-export const menuAuthorSelector: () => LiveSelector<E, true> = () =>
-    querySelector<E>(['div[style="height: 56px;"] a', '.GlobalNavigation a[href="/"]'].join(','))
-// export const entryDetailSelector: () => LiveSelector<E, true> = () => querySelector<E>('a[href="/"]')
-
 export const postsContentSelector = () =>
     querySelectorAll(
         [
             // In Entries
             '[id="__next"] > div:nth-child(2) > div > div:not([class]) > div:not(footer)',
             // In collection
-            '[id="__next"] > div:nth-child(2) > div > div > div > a:has(footer)',
+            '[id="__next"] > div:nth-child(2) a:has(footer)',
+            '[id="__next"] > div:nth-child(2) a:has(img[alt="Card Header"][loading="lazy"])',
```

Comment:
Attention i18n alternative text.

Reply:
1. Mirror doesn't have yet.
2. It's hardcoded
<img width="686" alt="image" src="https://github.com/DimensionDev/Maskbook/assets/1141198/22e881bf-2c71-46b7-aa83-0ae11035fd38">

Reply:
LOL if this is not the only selector, maybe it's ok